homworkW1.js

```diff
@@ -0,0 +1,47 @@
+/*snakewaterMontana = {
+    paleontologist: "Dr. Alan Grant",
+    depth: "10 meters",
+    specimen: "Velociraptor"
+  }
+  var guestOfHonor=snakewaterMontana.paleontologist;
+  console.log(guestOfHonor);
+  var cleverGirl=snakewaterMontana.specimen;
+  console.log(cleverGirl);*/
+
+  nicaragua = {
+    depth: "200 meters",
+    annualBudget: 1500000,
+    specimens: [
+      "Tyrannosaurus Rex",
+      "Stegosaurous",
+      "Triceratops",
+      "Velociraptor"
+    ]
+  }
+  var nicaraguanSpecimens=nicaragua.specimens;
+  /*console.log(nicaraguanSpecimens);*/
+
+  var favoriteSpecimen = nicaraguanSpecimens[1,2];
```

Comment:
To access a value inside an array we only use one index at a time so you can't access the element like this the correct syntax will be something like this ` var favoriteSpecimen = nicaraguanSpecimens[1];` or ` var favoriteSpecimen= nicaraguanSpecimens[2];`